app/views/reset_pins/change_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>Change PIN</h1>
+<%= form_tag change_pin_path, method: :post do %>
+  <%= hidden_field_tag :token, params['token'] %>
```

Comment:
This comes from the URL that is visited. Symphony when using the `changeMyPin` action, will verify that the correct token came through.